From 309c159fe0c91d56b42210bada221837ef0fa48e Mon Sep 17 00:00:00 2001
From: Shawn Guo <shawn.guo@freescale.com>
Date: Sun, 14 Jul 2013 21:10:37 +0800
Subject: [PATCH 0267/5242] MLK-11395-2: reset: build in CONFIG_RESET_GPIO by
 default

commit  ed3a4768e70a88a32d325c9fe884e65bacfe5288 from
https://source.codeaurora.org/external/imx/linux-imx.git

GPIO is widely used as the reset control for various devices.  Let's
build the support in by default.

[shawn.guo: cherry-pick commit 795fcb3bc5bb from imx_3.10.y]
Signed-off-by: Shawn Guo <shawn.guo@freescale.com>

(cherry picked from commit 0cbf78b5b02c57e6fd0e57e811cfe56509c4fd24)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/reset/Kconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/reset/Kconfig b/drivers/reset/Kconfig
index e4a73a6..fc712a5 100644
--- a/drivers/reset/Kconfig
+++ b/drivers/reset/Kconfig
@@ -63,6 +63,7 @@ config RESET_LANTIQ
 
 config RESET_GPIO
 	tristate "GPIO reset controller support"
+	default y
 	depends on GPIOLIB && OF
 	help
 	  This driver provides support for reset lines that are controlled
-- 
1.7.9.5

